From amit.kucheria@canonical.com  Mon Jul 27 09:37:38 2009
From: Amit Kucheria <amit.kucheria@canonical.com>
Date: Mon, 27 Jul 2009 12:03:19 +0300
Subject: USB: musb: Refer to musb_otg_timer_func under correct #ifdef
To: felipe.balbi@nokia.com
Cc: linux-usb@vger.kernel.org, gregkh@suse.de, Amit Kucheria <amit.kucheria@canonical.com>
Message-ID: <1248685399-25449-1-git-send-email-amit.kucheria@canonical.com>


From: Amit Kucheria <amit.kucheria@canonical.com>

musb_otg_timer_func() is defined under #ifdef CONFIG_USB_MUSB_OTG.
Make sure any reference to it is also under the same #ifdef.

Without this fix, the driver failes to compile when USB_OTG is defined
but USB_MUSB_OTG isn't.

Signed-off-by: Amit Kucheria <amit.kucheria@canonical.com>
Cc: Felipe Balbi <felipe.balbi@nokia.com>
Cc: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/musb_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/musb_core.c
+++ b/drivers/usb/musb/musb_core.c
@@ -1990,7 +1990,7 @@ bad_config:
 	if (status < 0)
 		goto fail2;
 
-#ifdef CONFIG_USB_OTG
+#ifdef CONFIG_USB_MUSB_OTG
 	setup_timer(&musb->otg_timer, musb_otg_timer_func, (unsigned long) musb);
 #endif
 
